packages/react-app/src/pages/Home.tsx

```diff
@@ -1,24 +1,36 @@
-import React from "react";
-import {BoxWrapper, BoxRow} from "../components";
+import React, { useState } from "react";
+import { BoxWrapper, BoxRow } from "../components";
 import Button from '@mui/material/Button';
 import Box from '@mui/material/Box';
-import {Link} from "react-router-dom";
-import {useTournaments} from "../hooks/useTournaments";
-import {Tournament} from "../graphql/subgraph";
-import {formatAmount, getTimeLeft} from "../lib/helpers";
+import { Link } from "react-router-dom";
+import { useTournaments } from "../hooks/useTournaments";
+import { Tournament } from "../graphql/subgraph";
+import { formatAmount, getTimeLeft } from "../lib/helpers";
+import { FormControlLabel, FormGroup, Switch } from "@mui/material";
 
 function Home() {
-  const { isLoading, error, data: tournaments } = useTournaments();
+  const [verifiedStatus, setVerifiedStatus] = useState<boolean>(false);
+  const { isLoading, error, data: tournaments } = useTournaments({ curated: verifiedStatus });
```

Comment:
`curated=false` shows only unverified tournaments, we want to use `curated=undefined` that shows both verified and unverified tournaments. You can change it to `useTournaments({ curated: verifiedStatus || undefined})`

Reply:
ups!, good catch!, fixing...